spyder/widgets/mixins.py

```diff
@@ -22,9 +22,11 @@
 from qtpy.QtCore import QPoint, Qt
 from qtpy.QtGui import QCursor, QTextCursor, QTextDocument
 from qtpy.QtWidgets import QApplication, QToolTip
-from qtpy import PYQT5
+from qtpy import PYQT_VERSION
 
-if PYQT5:
+PYQT_VERSION = float(PYQT_VERSION)
+

```

Comment:
Well.. sure, I meant using a single global thing, not how you were comparing versions (which, yes is probably wrong)

No need

You can compare tuples so use that
```python
a = (5, 5)
b = (5, 5, 1)

a > b
b < a
```